doc/api/url.markdown

```diff
@@ -72,7 +73,8 @@ Take a URL string, and return an object.
 Pass `true` as the second argument to also parse the query string using the
 `querystring` module. If `true` then the `query` property will always be
 assigned an object, and the `search` property will always be a (possibly
-empty) string.  Defaults to `false`.
+empty) string. If `false` then the neither the `query` nor the `search`
+properties will be parsed or decoded. Defaults to `false`.
```

Comment:
By looking at the code I'm not really sure search is parsed at all?

Not too familiar with `url` code though.